modules/virt-configuring-interface-link-state.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_networking/virt-setting-interface-link-state.adoc
+
+:_mod-docs-content-type: PROCEDURE                                  
+[id="virt-configuring-interface-link-state_{context}"]
+= Setting the VM interface link state by using the CLI
+
+You can set the link state of a primary or secondary virtual machine (VM) network interface by using the CLI.
```

Comment:
Please consider adding a short explanation about what this feature does:
It allows users to logically disconnect the vNIC from a network - it is the virtual equivalent of physically disconnecting an Ethernet cable from the physical NIC.